apps/site/src/views/Schedule/sections/ClipboardSchedule/ClipboardSchedule.tsx

```diff
@@ -27,21 +27,26 @@ const dateTimeFormat = new Intl.DateTimeFormat("en", {
 	minute: "numeric",
 });
 
+const weekdayFormat = new Intl.DateTimeFormat("en", {
+	weekday: "long",
+});
+
 function ClipboardSchedule({ schedule }: ClipboardScheduleProps) {
 	return (
 		<Container
 			as="section"
-			className={styles.clipboard + " px-0 position-relative"}
+			className={styles.clipboard + " px-0 pt-5 position-relative"}
 		>
 			<div className={styles.clip}>
 				<Image src={clip} alt="Clipboard clip" className={styles.clip} />
 			</div>
 			<Accordion defaultActiveKey="0" className={styles.accordion}>
 				{schedule.map((day, i) => (
-					<div key={i}>
-						<h2 className={styles.date}>
+					<div className="mt-5" key={i}>
+						<h2>{weekdayFormat.format(day[0].startTime)}</h2>
+						<h3 className="text-center mb-5">
 							{day[0].startTime.toLocaleDateString()}
```

Comment:
Suggestion [non-blocking]: Personally not a fan of the numeric formatting here, think the full month, day ordinal, and no year (November 4th) would look much nicer, but ordinal suffixes aren't directly supported by the locale formatting.

Reply:
I added the full month and day, hopefully that should be better here.